keps/sig-auth/1205-bound-service-account-tokens/kep.yaml

```diff
@@ -0,0 +1,28 @@
+---
+title: Bound Service Account Tokens
+authors:
+  - "@mikedanese"
+owning-sig: sig-auth
+participating-sigs:
+  - sig-auth
+reviewers:
+  - "@liggitt"
+approvers:
+  - "@liggitt"
+creation-date: 2019-08-06
+last-updated: 2020-07-29
+status: implemented
+feature-gates:
+  - name: TokenRequest
+    components:
+    - kube-apiserver
+    - kube-controller-manager
+  - name: TokenRequestProjection
+    components:
+    - kube-apiserver
+    - kubelet
+  - name: BoundServiceAccountTokenVolume
+    components:
+      - kube-apiserver
+      - kube-controller-manager
+---
```

Comment:
add the stage and milestone info (https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/kep.yaml#L26-L38) and metrics (https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/kep.yaml#L49-L51)

Reply:
done